src/client/app/redux/actions/conversions.ts

```diff
@@ -1,75 +1,40 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-import { ActionType, Thunk, Dispatch, GetState } from '../types/redux/actions';
-import { showSuccessNotification, showErrorNotification } from '../utils/notifications';
-import translate from '../utils/translate';
-import * as t from '../types/redux/conversions';
-import { conversionsApi } from '../utils/api';
+/* eslint-disable jsdoc/check-param-names */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+// @ts-nocheck
+/* eslint-disable jsdoc/require-param */
+
+// TODO Marked For Deletion after RTK migration solidified
```

Comment:
Not sure used and checking if you are okay with removing this now?

Reply:
Yep, Should go.